src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceRadio.vue

```diff
@@ -1,15 +1,35 @@
 <template>
-  <v-radio-group>
-    <v-radio label="Radio One" value="one"></v-radio>
-    <v-radio label="Radio Two" value="two"></v-radio>
-    <v-radio label="Radio Three" value="three"></v-radio>
+  <div>
+    <h3>
+    {{ props.title }}
+  </h3>
+  <br />
+  <v-radio-group :rules="[Rules.requiredRadio()]" @update:model-value="(value) => $emit('update:model-value', value as string)">
+    <v-radio v-for="option in props.options" :key="option.key" :label="option.label" :value="option.key"></v-radio>
   </v-radio-group>
+  </div>
 </template>
 
 <script lang="ts">
 import { defineComponent } from "vue";
+import * as Rules from "@/utils/formRules";
+import type { EceRadioFieldProps } from "@/types/input";
 
 export default defineComponent({
   name: "EceRadio",
+  props: {
+    props: {
+      type: Object as () => EceRadioFieldProps,
+      required: true,
+    },
+  },
+  emits: {
+    "update:model-value": (_reference: string) => "",
```

Comment:
`"update:model-value": (_value: string) => "",` I assume this is from another component?